agent/bin/ci.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+# Starts a Cocoon agent.
+
+USER_NAME="flutter"
+CONFIG_FILE="config.yaml"
+
+if [ "$(whoami)" != "$USER_NAME" ]; then
```

Comment:
I think this script will only work for mac and linux how are we going to handle windows?

Reply:
We will add windows script later.